src/app/core/services/helper.service.ts

```diff
@@ -192,6 +193,13 @@ export class HelperService {
     return urlParams.get(name);
   }
 
+  addUrlParam(url: string, paramName: string, paramValue: string | number | boolean) {
```

Comment:
this method adds the ability to generate a url with a single parameter (so we can support adding the 'model' url param)

Reply:
Makes sense! What do you think about adding a comment to capture this constraint or updating the name, e.g. `addSingleUrlParam`?

Reply:
good suggestion.  I will rename it.